docs/Feature_Implementation_Matrix.md

```diff
@@ -0,0 +1,172 @@
+# Trillian Feature/Implementation Matrix
+
+ - [Overview](#overview)
+ - [Functionality](#functionality)
+   - [Log v1](#trillian-log-v1)
+   - [Log v2](#trillian-log-v2-skylog)
+   - [Map](#trillian-map)
+   - [Log backed map](#log-backed-map)
+ - [Concrete implementations](#concrete-implementations)
+   - [Storage](#storage)
+   - [Monitoring](#monitoring)
+   - [Master election](#master-election)
+   - [Quota](#quota)
+
+
+## Overview
+
+This page summarises the various features which are present in Trillian, and
+their implementation status.
+
+The status of features is listed as one of:
+  * Not Implemented (NI)
+  * In Development
+  * Partial
+  * Alpha
+  * Beta
+  * General Availability (GA)
+  * Deprecated (⚠)
+
+
+## Functionality
```

Comment:
The vertical bars in (all of) your tables don't really line up. Your call on how much time you want to spend on this - it looks fine in the rendered view.

Reply:
tabs vs. spaces, the eternal battle :)